src/tools/abstracttwopointtool.cpp

```diff
@@ -36,6 +36,19 @@ AbstractTwoPointTool::AbstractTwoPointTool(QObject* parent)
   , m_padding(0)
 {}
 
+void AbstractTwoPointTool::copyParams(const AbstractTwoPointTool* from,
```

Comment:
Might be cleaner to replace with assignment operator? Just a suggestion, not important.

Reply:
I don't exactly remember why did I do it in this way. I had some difficulties with "operator=" and this was was easier. I used it in another similar case, but not here. If it's critical, I'll refactor it.

Reply:
I'll take a look at it and will do refactor later, I have noted it to myself.